cwltool/load_tool.py

```diff
@@ -124,6 +124,8 @@ def _convert_stdstreams_to_files(workflowobj):
     if isinstance(workflowobj, dict):
         if workflowobj.get('class') == 'CommandLineTool':
             for out in workflowobj.get('outputs', []):
+                if type(out) is not CommentedMap:
+                    raise ValidationException("Output '%s' is not valid" % out)
```

Comment:
the error message becomes:
```
$ cwltool --debug https://raw.githubusercontent.com/EOSC-LOFAR/prefactor-cwl/c06f92eb9cea4ab34cfffcbcc48c5d32aad2a1a7/ndppp_prep_cal.cwl --help
/home/michael/cwltool/env/bin/cwltool 1.0.20180201074953
Tool definition failed validation:
Output 'None' is not valid
Traceback (most recent call last):
  File "/home/michael/cwltool/cwltool/main.py", line 873, in main
    overrides=overrides)
  File "/home/michael/cwltool/cwltool/load_tool.py", line 269, in validate_document
    _convert_stdstreams_to_files(workflowobj)
  File "/home/michael/cwltool/cwltool/load_tool.py", line 128, in _convert_stdstreams_to_files
    raise ValidationException("Output '%s' is not valid" % out)
ValidationException: Output 'None' is not valid
```
which is still not helpful to the user, can you add the line number using the same technique from https://github.com/common-workflow-language/cwltool/pull/635/files#diff-bfbf952b5c2585de423b12dcf6f43938R210 ?